clients/src/test/java/org/apache/kafka/common/network/SslSelectorTest.java

```diff
@@ -125,10 +125,17 @@ public void testConnectionWithCustomKeyManager() throws Exception {
         selector.send(createSend(node, request));
 
         waitForBytesBuffered(selector, node);
+        assertEquals(1, metrics.metrics().keySet().stream().
+            filter(n -> n.description().contains("The number of connections with this SSL cipher and protocol.")).
+            count());
```

Comment:
We could also check the value of that metric.

Reply:
done